### PR TITLE
Feature/rename file on remote server

### PR DESCRIPTION
**Pull request by carissaallen**
Thanks for your review! 🌮

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -137,7 +139,18 @@ private static void rename(Client client) throws SftpException {
           break;
         case 6:
           System.out.println("Rename remote directory/file...");
+          try {
+            client.displayRemoteFiles();
+            out.println("Enter the original file name: ");
```
Comment (rooneyshuman) [inline src/main/java/Main.java]: To keep the switch statement clean and for the sake of consistency (change dir and create dir methods prompt from within), I'd move the filename prompts to the method implementation in the client class and just call the method here without args. Other than that, it works perfectly!

Comment (carissaallen) [reply src/main/java/Main.java]: Perfect, thanks! 👌

**Comment by carissaallen** [comment]
I am rewriting this function so it's testable... 💩

**Comment by rooneyshuman** [comment]
Just implemented an "overwrite" feature to my createRemoteDir method in the branch overwrite_in_createDirs if you want to take a look and do something similar. I noticed we weren't accounting for the fact that the user could create/rename directories with a directory name that already exists. 👍

**Pull request closed without merging**